lib/qa/authorities/web_service_base.rb

```diff
@@ -11,9 +11,15 @@ def get_json(url)
     end
 
     def response(url)
-      Faraday.get(url) do |req|
-        req.headers['Accept'] = 'application/json'
+      uri = URI(url)
+      conn = Faraday.new uri.scheme+"://"+uri.host
+      conn.options.params_encoder = Faraday::FlatParamsEncoder
```

Comment:
That does make sense (although I personally think that Faraday's default behavior is silly in this case). I have updated the pull request based on your suggestion.